### ISSUE TITLE
Output "Component" level of archives

### ISSUE THREAD

**Issue by TheRogueArchivist** [mnadareski/UnshieldSharp#8]
Some (most?) archives contain internal folders, the contents of which are just currently all extracted directly into the output directory. This is especially an issue in cases where files with the same file name that are separated into separate internal directories. These are not currently being extracted by UnshieldSharp due to them being files with the same name being extracted into the same directory.

**Comment by TheRogueArchivist** [comment]
I just discovered that what I specifically want here appears to be the component level of archives to be output as well, as they contain useful information and context about the files that are extracted. It will likely also make it easier for me to investigate and improve specific DRM scans in BOS.

**Comment by mnadareski** [comment]
I was wrong with my assumption. Components are _not_ actually included in the output path by default in Unshield. However, file groups _are_ included. See the following code for how it determines this:

```cpp
for (i = 0; i < unshield_file_group_count(unshield); i++)
{
    UnshieldFileGroup* file_group = unshield_file_group_get(unshield, i);
    if (file_group)
        count += helper(unshield, file_group->name, file_group->first_file, file_group->last_file);
}
```

**Comment by TheRogueArchivist** [comment]
Seems to be fixed by bb9cd947b0f5b32d9d01ae2147c31bc9d8d9b530.
